Chaincase.Common/Global.cs

```diff
@@ -229,6 +232,11 @@ public async Task InitializeNoWalletAsync(CancellationToken cancellationToken =
 
                 _walletManager.RegisterServices(_bitcoinStore, _synchronizer, Nodes, _config.ServiceConfiguration, _feeProviders, blockProvider);
 
+                if (!_P2EPServer.HiddenServiceIsOn) {
+                    var cts = new CancellationToken();
+                    await _P2EPServer.StartAsync(cts);
```

Comment:
Why the conditional? The synchronizer manages it's own lifecycle so if you button mash it doesn't cause problems. I think P2EPServer's lifecycle should be self contained and not reliant on external state checks